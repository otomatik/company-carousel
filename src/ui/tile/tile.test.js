import { shallow } from 'enzyme';
import React from 'react';
import { Tile, LinkTile, PhotoTile, QuoteTile, VideoTile } from './';

describe('Tile', () => {
  it('should match snapshot', () => {
    // when
    const tile = shallow(<Tile color="yellow">tuile</Tile>);

    // then
    expect(tile).toMatchSnapshot();
  });
});

describe('LinkTile', () => {
  it('should match snapshot', () => {
    // when
    const tile = shallow(
      <LinkTile url="http://" color="yellow">
        tuile
      </LinkTile>
    );

    // then
    expect(tile).toMatchSnapshot();
  });
});

describe('QuoteTile', () => {
  it('should match snapshot', () => {
    // when
    const tile = shallow(<QuoteTile text="coucou" />);

    // then
    expect(tile).toMatchSnapshot();
  });
});

describe('VideoTile', () => {
  it('should match snapshot', () => {
    // when
    const tile = shallow(<VideoTile url="http://" thumbnail="http://" />);

    // then
    expect(tile).toMatchSnapshot();
  });
});

describe('VideoTile', () => {
  it('should match snapshot', () => {
    // when
    const tile = shallow(<PhotoTile url="http://" thumbnail="http://" />);

    // then
    expect(tile).toMatchSnapshot();
  });
});
