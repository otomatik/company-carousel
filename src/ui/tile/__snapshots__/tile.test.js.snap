// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkTile should match snapshot 1`] = `
<styled.a
  href="http://"
  target="_blank"
>
  <Tile
    color="yellow"
  >
    tuile
  </Tile>
</styled.a>
`;

exports[`QuoteTile should match snapshot 1`] = `
<Tile
  color="#444"
>
  <styled.blockquote>
    coucou
  </styled.blockquote>
</Tile>
`;

exports[`Tile should match snapshot 1`] = `
<styled.div
  color="yellow"
>
  tuile
</styled.div>
`;

exports[`VideoTile should match snapshot 1`] = `
<LinkTile
  url="http://"
>
  <styled.div
    thumbnail="http://"
  />
  <styled.div>
    <Play />
    <styled.div>
      <styled.p />
      <Styled(styled.p) />
    </styled.div>
  </styled.div>
</LinkTile>
`;

exports[`VideoTile should match snapshot 2`] = `
<LinkTile
  url="http://"
>
  <styled.div
    thumbnail="http://"
  />
  <styled.div>
    <Styled(More) />
  </styled.div>
</LinkTile>
`;
